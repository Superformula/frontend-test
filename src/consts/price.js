const PRICE_OPTIONS = Object.freeze(['$', '$$', '$$$', '$$$$']);

export const PRICES = PRICE_OPTIONS.map(price => ({
  label: price,
  value: price,
}));
