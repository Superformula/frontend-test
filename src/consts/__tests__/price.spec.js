import { PRICES } from '../price';

describe('PRICES', () => {
  it('should have proper values', () => {
    expect(PRICES).toMatchSnapshot();
  });
});
