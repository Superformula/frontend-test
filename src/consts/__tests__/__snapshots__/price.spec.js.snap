// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PRICES should have proper values 1`] = `
Array [
  Object {
    "label": "$",
    "value": "$",
  },
  Object {
    "label": "$$",
    "value": "$$",
  },
  Object {
    "label": "$$$",
    "value": "$$$",
  },
  Object {
    "label": "$$$$",
    "value": "$$$$",
  },
]
`;
